docs/usage/ExcludeResources.md

```diff
@@ -0,0 +1,67 @@
+# Resource Exclusion
+Starting with CAM 1.2.4 it is possible to exclude PV data migration, ImageStream migration, and migration of specific Kubernetes resources kinds. This can be done to allow migration of data or images with an alternative tool or to prevent migration of problematic kubernetes resources.
+
+# Exclude ImageStream Migration
+Modify the MigrationController CR and add `disable_image_migration: true` to the spec section.
+
+```
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigrationController
+metadata:
+  name: migration-controller
+  namespace: openshift-migration
+spec:
+  disable_image_migration: true
+...
+```
+
+# Exclude PV Migration
+Modify the MigrationController CR and add `disable_pv_migration: true` to the spec section.
+
+```
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigrationController
+metadata:
+  name: migration-controller
+  namespace: openshift-migration
+spec:
+  disable_pv_migration: true
```

Comment:
@jmontleon would you please add more comments that when "disable_pv_migration: true", it won't discover PVs while creating migration plan?

Reply:
This got merged before your request. Adding a note in a separate PR: https://github.com/konveyor/mig-operator/pull/394/files